src/servers/http_server_v2.cc

```diff
@@ -998,27 +1010,21 @@ HTTPAPIServerV2::HandleModelMetadata(
 
     evbuffer_add(
         req->buffer_out, model_metadata.c_str(), model_metadata.size());
+    evhtp_send_reply(req, EVHTP_RES_OK);
+  } else {
+    document.Clear();
```

Comment:
Should make a common routine for this, don't copy-paste. Also this seems unnecessarily complex for such a simple json response. Just create it as a string... will be better performance also.